pkg/controller/migmigration/backup.go

```diff
@@ -305,17 +382,23 @@ func (t *Task) annotateStorageResources() error {
 		client.Update(context.TODO(), &resource)
 	}
 
+	resticPodCount := 0
 	for _, ns := range namespaces {
 		// Find all pods in our target namespaces
 		list := corev1.PodList{}
 		options := k8sclient.InNamespace(ns)
 		err = client.List(context.TODO(), options, &list)
 		if err != nil {
 			log.Trace(err)
-			return err
+			return err, 0
 		}
 		// Loop through all pods to find all volume claims
 		for _, pod := range list.Items {
+			// Don't need to do this for staging pods. Adds unnecessary work
+			if pod.Labels[fmt.Sprintf("%s-copy", uniqueBackupLabelKey)] != "" {
```

Comment:
Do we need to check that `pod.Labels` isn't nil before checking a value in its map?